packages/devtools/lib/src/utils.dart

```diff
@@ -105,6 +105,16 @@ String funcRefName(FuncRef ref) {
   }
 }
 
+void maybeExecuteWithDelay(bool condition, Duration delay, void callback()) {
```

Comment:
maybeExecuteWithDelay -> executeWithDelay

maybe change condition to executeNow and make it an optional named parameter?